src/microbe_stage/PatchMap.cs

```diff
@@ -320,4 +370,127 @@ public void ReplaceSpecies(Species old, Species newSpecies)
             patch.ReplaceSpecies(old, newSpecies);
         }
     }
+
+    public void SetSeed(Random random)
```

Comment:
Setter for a **public** field is super useless as people can just do the incorrect thing and modify it directly.